cowprotocol/cowswap
# Added GA dimension for wallet name

# Summary

Part of the #665

- This PR adds GA dimension for wallet name

### To test
- have installed https://chrome.google.com/webstore/detail/google-analytics-debugger/jnkmfdileelhofjcijamephohjechhna?hl=en or something similar
- change wallet or chain
- you should see something like `Processing GTAG command: ["set", {walletName: "MetaMask"}]` in your console
